apps/backend/package.json

```diff
@@ -27,6 +27,7 @@
     "crypto-random-string": "^5.0.0",
     "dotenv": "^16.4.5",
     "express": "^4.19.2",
+    "express-session": "^1.18.1",
```

Comment:
This doesn't seem to be used anywhere in the app.

Reply:
yes ,I will remove it